packages/manager/modules/bm-server-components/src/order-private-bandwidth/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,106 @@
+{
+  "server_error_bandwidth_order": "Une erreur est survenue lors de la commande de bande passante : {{error}}",
+  "server_order_bandwidth_title": "Commander de la bande passante",
+  "server_order_bandwidth_vrack_title": "Commander de la bande passante",
+  "server_order_bandwidth_not_available": "Aucun type de bande passante n'est commandable sur le serveur {{t0}}",
+  "server_order_bandwidth_help": "Les offres Premium, Platinum et Ultimate garantissent la qualité de la bande passante sur une zone géographique plus ou moins étendue jusqu'au plus près de vos clients.",
+  "server_order_bandwidth_ready": "Votre commande est prête. Sous réserve de paiement, votre bande passante devrait être disponible prochainement.",
+  "server_order_bandwidth_see_order": "Voir le bon de commande",
+  "server_order_bandwidth_vrack_load": "Chargement des offres en cours...",
+  "server_order_bandwidth_empty_plans": "Impossible de trouver des plans pour mettre à niveau",
+  "select_bandwidth_type_label": "Sélectionnez une offre",
+  "select_bandwidth_vrack_type_label": "Sélectionnez une offre",
+  "select_bandwidth_plan_summary": "Récapitulatif de la commande",
+  "server_order_bandwidth_vrack_plan_text": "de bande passante privée, trafic illimité (vRack)",
```

Comment:
This sentance seems weired.
i think we forgot some part.

Reply:
Good observation. Its a legacy text messages. Looks like its not used. I dont want to remove this as part of the refactoring. It can be taken as a separate task.